### ISSUE TITLE
'findDOMNode' is not exported from 'react-dom' (imported as 'findDOMNode').

### ISSUE THREAD

**Issue by megetron** [rsuite/rsuite#3248]
### What version of rsuite are you using?

5.35.1

### What version of React are you using?

18.2.0

### What version of TypeScript are you using (if any)?

5.1.3

### What browser are you using?

chrome

### Describe the Bug

when importing rsuite and using the TagGroup and Tag components, then i get error when close a tag (pressing on 'x'):
'findDOMNode' is not exported from 'react-dom' (imported as 'findDOMNode').

### Expected Behavior

no error should be displayed.

### To Reproduce

just install the above versions and create taggroup and closable tags. 
close a tag and get the error.

to fix the issue on my side i opened:
vi ./node_modules/.pnpm/rsuite@5.35.1_biqbaboplfbrettd7655fr4n2y/node_modules/rsuite/esm/utils/getDOMNode.js
import ReactDOM from ‘react-dom’;
instead of
import {ReactDOM} from ‘react-dom’;

**Comment by SevenOutman** [comment]
Hi @megetron, I tried closable Tags in [this CodeSandbox](https://codesandbox.io/s/naughty-wind-vk7wr8?file=/src/index.js) couldn't re-produce the problem you described. Could you provide a CodeSandbox or a demo repo for reproduction?

**Comment by minhktmd** [comment]
I have same problem with my nextjs project when I try using some rsuite components like (Loader, SelectPicker, Drawer)
It's a warning, not an error but it is so annoying

- warn ./node_modules/rsuite/esm/utils/getDOMNode.js
Attempted import error: 'findDOMNode' is not exported from 'react-dom' (imported as 'findDOMNode').

Import trace for requested module:
./node_modules/rsuite/esm/utils/getDOMNode.js
./node_modules/rsuite/esm/utils/index.js
./node_modules/rsuite/esm/Button/Button.js
./node_modules/rsuite/esm/Button/index.js
./node_modules/rsuite/esm/index.js

**Comment by simonguo** [comment]
Hi  @megetron @minhktmd , I am unable to reproduce the issue locally, so I've made a preventative fix.
Please install a temporary version of rsuite using the following command and check if this resolves the issue.



```
npm i https://pkg.csb.dev/rsuite/rsuite/commit/4656cae1/rsuite
```

**Comment by minhktmd** [comment]
Hi @simonguo 
I installed the temporary version of rsuite you give above and the problem has been resolved.
Thanks for your support, hope this update will be avaiable in next offcial version of rsuite

**Comment by simonguo** [comment]
Fixed in [rsuite@5.41.0](https://github.com/rsuite/rsuite/releases/tag/v5.41.0)
